### PR TITLE
Separate worker depends

### PR DESCRIPTION
**Pull request by banesullivan**

### CODE REVIEW

**Review by brianhelba — COMMENTED** [review]
https://github.com/ResonantGeoData/ResonantGeoData/blob/master/dev/django.Dockerfile#L28 too

File: setup.py [hunk 1]
```diff
@@ -39,12 +39,16 @@
...
-        'rasterio',
-        'fiona',
-        'shapely',
-        'scipy',
-        'kwcoco',
-        'kwimage',
+        'numpy',
     ],
-    extras_require={'dev': ['ipython', 'tox']},
+    extras_require={'dev': ['ipython', 'tox'],
```
Comment (brianhelba) [inline setup.py:44]: Need to run `tox -e format` to fix this.

**Comment by brianhelba** [comment]
Rebase this on `master` too, which will fix failing tests.

**Comment by banesullivan** [comment]
Looks like rest_framework may have changed? 

```
from rest_framework.compat import URLPattern, URLResolver, get_original_route
E   ImportError: cannot import name 'URLPattern' from 'rest_framework.compat' (/home/circleci/project/.tox/test/lib/python3.8/site-packages/rest_framework/compat.py)
```

https://app.circleci.com/pipelines/github/ResonantGeoData/ResonantGeoData/547/workflows/5dc4c245-3a57-4c3b-a12f-abbcccb6677b/jobs/552/parallel-runs/0/steps/0-103

version 3.12.x of rest framework was released yesterday https://github.com/encode/django-rest-framework/releases

**Comment by banesullivan** [comment]
I think this is ready to go, @brianhelba

**Review by brianhelba — APPROVED** [review]

**Pull request merged**